src/components/Form.js

```diff
@@ -0,0 +1,88 @@
+import React, { Fragment, Component } from 'react';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import { logIn } from '../services/userService';
+import { Redirect } from 'react-router-dom';
+import '../App.css';
+
+
+export default class Form extends Component {
+	state = {
+		login: '',
+		password: '',
+		notification: '',
+		notificationType: '',
+		redirect: false
+	}
+
+	handleChangeInput = e => {
+		this.props.onChange({[e.target.name]: e.target.value});
+		this.setState({[e.target.name]: e.target.value});
+	}
+
+	handleSubmit = e => {
+		e.preventDefault();
+		const { login, password } = this.state;
+		logIn({ login, password })
+								.then(result => {
+									this.setState({notification: result, notificationType: 'notification success' });
+									if(result) {
+										this.setState({redirect: true});
+									}
+								})
+								.catch(error => {
+									this.setState({notification: error, notificationType: 'notification error' });
+								});
+
+		this.showNotification = (notification) => {
+			this.setState({notification});
+
+			setTimeout(() => {
+				this.setState({notification: null})
```

Comment:
this.setState({ notification: null }); Do not forget spaces here.

Reply:
done